File: Character_Count/character_count.py
#take user input
String = input('Enter the string :')
#take character input
Character = input('Enter character :')
#initiaalize int variable to store frequency
frequency = 0
#use count function to count frequency of character
frequency = String.count(Character)
#count function is case sencetive 
#so it print frequency of Character according to given Character
print(str(frequency) + ' is the frequency of given character')

File: Character_Count/character_count.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    // Initializing variables.
    string str;
    int freq[256] = {0};
    
    // Accepting input.
    cout << "Enter the string: ";
    getline(cin, str);
    
    // Calculating frequency of each character.
    for (char ch : str)
    {
        freq[static_cast<unsigned char>(ch)]++;
    }
    
    // Printing frequency of each character.
    for (int i = 0; i < 256; i++)
    {
        if (freq[i] != 0)
        {
           cout << "The frequency of " << char(i) << " is " << freq[i] << endl;
        }
    }
    return 0;
}
